src/Main.java

```diff
@@ -2,14 +2,101 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-        // to see how IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+        int    a=13567008;
+        byte   b=127;
+        short  c=-30000;
+        long   d=5609090L;
+        float  e=5.987f;
+        double f=-9.090091234;
+
+        float  g=27.12f;
+        long   h=987678965549L;
+        char   i=2786;
+        short  k=569;
+        short  l=-159;
+        int    m=27897;
+        short  n=67;
+
+        byte  studentsLudmilaPavlovna=23;
+        byte  studentsAnnaSergeevna=27;
+        byte  studentsEkaterinaAndreevna=30;
+        int   listOfPapers=480;
+        int   listPerStudent=listOfPapers/(studentsLudmilaPavlovna+studentsAnnaSergeevna+studentsEkaterinaAndreevna);
+        System.out.println("На каждого ученика рассчитано "+listPerStudent+" листов бумаги.");
+        System.out.println("   ");
+
+        byte  outputPerMinute=16/2;
+        int   outputPer20Minute=outputPerMinute*20;
+        int   outputPerDay=outputPerMinute*24*60;
+        int   outputPer3Days=outputPerMinute*24*60*3;
```

Comment:
```suggestion
        int   outputPer3Days=3 * outputPerDay;
```
кол-во за 3 дня - это 3 * кол-ва за 1 день